Sources/TuistGenerator/Models/InfoPlist.swift

```diff
@@ -2,21 +2,70 @@ import Basic
 import Foundation
 import TuistCore
 
-public enum InfoPlist: Equatable, ExpressibleByStringLiteral, ExpressibleByUnicodeScalarLiteral, ExpressibleByExtendedGraphemeClusterLiteral {
+public enum InfoPlist: Equatable {
+    public indirect enum Value: Decodable, Equatable {
+        case string(String)
+        case integer(Int)
+        case boolean(Bool)
+        case dictionary([String: Value])
+        case array([Value])
+
+        public init(from decoder: Decoder) throws {
+            if let singleValueContainer = try? decoder.singleValueContainer() {
```

Comment:
This code is duplicated, is that because of Generator & ProjectDescription needing their own versions?

Reply:
Yes